logging.properties

```diff
@@ -1,3 +1,3 @@
-#jmri.jmrit.display.layoutEditor.level=DEBUG
-#jmri.jmrit.display.layoutEditor.LayoutBlockManager.level=DEBUG
-#jmri.jmrit.display.layoutEditor.LayoutBlock.level=DEBUG
+jmri.jmrit.display.layoutEditor.level=TRACE
```

Comment:
@geowar1 I think that the complete file itself should be removed.

Looks like it was inadvertently introduced as part of #2934